Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM centos:7
+FROM centos:7 as cfgov-runtime
 
-# Install required packages
+# Install build-required packages
 RUN yum install -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-centos10-10-2.noarch.rpm && \
-    yum install -y epel-release make gcc gcc-c++ kernel-devel mailcap which && \
-    yum install -y postgresql10 postgresql10-devel && \
-    yum install -y python-devel python36 python36-devel && \
-    yum clean all
+    yum install -y epel-release mailcap which httpd && \
+    yum install -y postgresql10  && \
+    yum install -y python36 && \
+    yum clean all && rm -rf /var/cache/yum
+
+ENV PYTHON /usr/bin/python3
+ENV PYTHONPATH=/src/cfgov-refresh/cfgov/
+COPY docker/python/entrypoint.sh /usr/local/bin/entrypoint.sh
+RUN mkdir /active-python/ && chown -R apache:apache /active-python
+
+ENTRYPOINT ["sh", "/usr/local/bin/entrypoint.sh"]
+CMD ["sh", "-c", "/active-python/python /src/cfgov-refresh/cfgov/manage.py runmodwsgi --port 8000 --user apache --group apache --log-to-terminal --working-directory /src/cfgov-refresh/  $EXTRA_MODWSGI_ARGS"]
+EXPOSE 8000
+
+
+FROM cfgov-runtime AS cfgov-dev
+
+# Install build-required packages
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn make gcc gcc-c++ kernel-devel unzip && \
```

Comment:
Do you need nodejs and yarn here with the commands on lines 23 and 24?

Reply:
As an expansion on this, we can make a choice here on whether we want node and yarn in the dev image at all. If users will be running yarn commands outside of the container with the results mapped in via the volume (which I think is the intended workflow?), then node and yarn aren't needed here. Of course, we'd then have to install them in the build image, but this would limit the size of the dev image.

Then again, we can just tell people to only use yarn commands *in* the containers, and tell people they no longer need to worry about node/yarn on there systems. This would come with some dev friction, performance cost, and headache around ensuring `static_built` on the local machine isn't mapping over container copies (in practice this should work fine, but there can be footguns).

Reply:
those lines set up the repos, I think they still need to be yum installed.

On the other question, 🤷‍♂?  You're right, if nobody will use yarn and node in the dev image, it could be punted to the build.